packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -493,3 +489,15 @@ function _tagMetricInfo(transaction: Transaction): void {
     );
   }
 }
+
+function setResourceEntrySizeData(
+  data: Record<string, unknown>,
+  entry: ResourceEntry,
+  key: keyof Pick<ResourceEntry, 'transferSize' | 'encodedBodySize' | 'decodedBodySize'>,
+  dataKey: 'http.response_transfer_size' | 'http.response_content_length' | 'http.decoded_response_content_length',
+): void {
+  const entryVal = entry[key];
+  if (entryVal !== undefined && entryVal < ONE_GB_IN_BYTES) {
+    data[dataKey] = entryVal;
+  }
```

Comment:
Will this one gb limit affect resources, such as `resource.video`, that could (I think), be larger then 1gb

Reply:
👍